kpatch-build/create-diff-object.c

```diff
@@ -1541,22 +1540,27 @@ static void kpatch_replace_sections_syms(struct kpatch_elf *kelf)
 				continue;
 			}
 
-#ifdef __powerpc64__
-			add_off = 0;
-#else
-			if (rela->type == R_X86_64_PC32 ||
-			    rela->type == R_X86_64_PLT32) {
-				struct insn insn;
-				rela_insn(sec, rela, &insn);
-				add_off = (unsigned int)((long)insn.next_byte -
-					  (long)sec->base->data->d_buf -
-					  rela->offset);
-			} else if (rela->type == R_X86_64_64 ||
-				   rela->type == R_X86_64_32S)
-				add_off = 0;
-			else
-				continue;
-#endif
+			switch(kelf->target_arch) {
+				case PPC64:
```

Comment:
The kernel style is to put 'case' on the same indentation level as 'switch', which helps reduce excessive indentation

Reply:
Is it?  I've observed 50/50 across the kernel, and in fact create-diff-object.c as well :P  It looks weird when the last case  adds it's own bracketed block like:
```C
switch(foo) {
case 1:
    /* code */
case 99: {
    /* code */
}                          <- appears unbalanced
}
```
but none of my redundant default cases introduce a local scope, so that never happens here.

Anyway, it's not a big deal, I'll update to help the greater indentation cause.

Reply:
> Is it? I've observed 50/50 across the kernel

Really?  `git grep -A1 switch` seems to show otherwise.

> and in fact create-diff-object.c as well :P

I'm not surprised by that, but it's never too late to change :-)

> It looks weird when the last case adds it's own bracketed block like:
> 
> ```c
> switch(foo) {
> case 1:
>     /* code */
> case 99: {
>     /* code */
> }                          <- appears unbalanced
> }
> ```

Ew, hopefully we can avoid the need for that.


> Anyway, it's not a big deal, I'll update to help the greater indentation cause.

Thanks :-)

Reply:
> Ew, hopefully we can avoid the need for that.

I see now that we did need it.  Looks reasonable though :-)